tidb-cloud/integrate-tidbcloud-with-vercel.md

```diff
@@ -34,63 +39,117 @@ One Vercel project can only connect to one TiDB Cloud cluster. To change the int
 
 ## Connect via the TiDB Cloud Vercel integration
 
 To connect via the TiDB Cloud Vercel integration, go to the [TiDB Cloud integration](https://vercel.com/integrations/tidb-cloud) page from the [Vercel's Integrations Marketplace](https://vercel.com/integrations). Using this method, you can choose which cluster to connect to, and TiDB Cloud will automatically generate all the necessary environment variables for your Vercel projects.
 
 The detailed steps are as follows:
 
+<SimpleTab>
+<div label="Direct connection">
+
 1. Click **Add Integration** in the upper-right area of the [TiDB Cloud Vercel integration](https://vercel.com/integrations/tidb-cloud) page. The **Add TiDB Cloud** dialog is displayed.
-2. Select the scope of your integration in the drop-down list and click **CONTINUE**.
-3. Select the Vercel Projects to which the integration will be added and click **CONTINUE**.
-4. Confirm the required permissions for integration and click **ADD INTEGRATION**. Then you are directed to an integration page of the TiDB Cloud console.
-5. On the left side, select the target Vercel projects and framework your Vercel projects using. If the framework isn't listed, select **General**. Different frameworks determine different environment variables.
-6. On the right side, select the target TiDB Cloud cluster after providing the cluster information. Each TiDB Cloud cluster belongs to [an organization and a project](/tidb-cloud/manage-user-access.md#organizations-and-projects).
-7. Click **Add Integration and Return to Vercel**.
+2. Select the scope of your integration in the drop-down list and click **Continue**.
+3. Select the Vercel projects to which the integration will be added and click **Continue**.
+4. Confirm the required permissions for integration and click **Add Integration**. Then you are directed to an integration page of the TiDB Cloud console.
+5. On the integration page, do the following:
 
-![Vercel Integration Page](/media/tidb-cloud/integration-vercel-link-page.png)
+    1. Select your target Vercel projects and click **Next**.
+    2. Select your target TiDB Cloud organization and project.
+    3. Select **Cluster** as your connection type.
+    4. Select your target TiDB Cloud cluster. If the **Cluster** drop-down list is empty or you want to select a new TiDB Serverless cluster, click **+ Create Cluster** in the list to create one.
+    5. Select the framework that your Vercel projects are using. If the target framework is not listed, select **General**. Different frameworks determine different environment variables.
+    6. Click **Add Integration and Return to Vercel**.
 
-8. Back to your Vercel dashboard, go to your Vercel project, click **Settings** > **Environment Variables**, and confirm that the environment variables have been automatically added.
+![Vercel Integration Page](/media/tidb-cloud/vercel/integration-link-cluster-page.png)
```

Comment:
This image should also be indented by four spaces, but I'm not sure whether it can display well now, considering a recent error. @qiancai What do you think?